bench/workgen/workgen.cxx

```diff
@@ -111,10 +111,11 @@ struct WorkloadRunnerConnection {
     WT_CONNECTION *connection;
```

Comment:
Sidenote: Does anyone know if there's a WT ticket to change the last of our `.cxx` files to `.cpp`?

Reply:
I'll do it after we merge this change.